### ISSUE TITLE
Option to pick how many mangas to download

### ISSUE THREAD

**Issue by zPilax** [KotatsuApp/Kotatsu#550]
### Describe your suggested feature

Currently on the latest update, all the mangas that you will download is downloaded all at the same time.

I would like to suggest a limit on concurrent downloads, so that we wont be blocked by the api requests that are being done while we are downloading.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
